### DISCUSSION TITLE
can't open any stream

### DISCUSSION THREAD

**Discussion by Summeromanceq** [streamlink/streamlink#4137] [Q&A] [answered]
Usage: streamlink https://www.twitch.tv/dreadztv best
Get : [cli][info] Found matching plugin twitch for URL https://www.twitch.tv/dreadztv
error: Unable to open URL: https://tmi.twitch.tv/hosts (404 Client Error: Not Found for url: https://tmi.twitch.tv/hosts?as3=t&include_logins=1&host=31089858)

**Comment by gravyboat** [comment]
Not much we can do with the information you have provided. Provide the full log output with `-l debug` on the command. I don't get the same output when I run the command you've provided and it seems like you're running some sort of unique configuration.

**Comment by bastimeyer** [comment, marked as answer]
> error: Unable to open URL: [tmi.twitch.tv/hosts](https://tmi.twitch.tv/hosts) (404 Client Error: Not Found for url: [tmi.twitch.tv/hosts?as3=t&include_logins=1&host=31089858](https://tmi.twitch.tv/hosts?as3=t&include_logins=1&host=31089858))

You are using an old version of Streamlink. The Twitch plugin doesn't query `tmi.twitch.tv` anymore since version `2.2.0` (changed in #3776)
